src/case.ts

```diff
@@ -32,3 +33,7 @@ export function capitalize(s: string): string {
 export function camelCase(s: string): string {
   return s.substring(0, 1).toLowerCase() + s.substring(1);
 }
+
+export function camelCaseGrpc(s: string): string {
```

Comment:
Cool, I like wrapping the `lodashCamelCase` in this `camelCaseGrpc` as a call out that we're doing this specifically to match the GRPC semantics.

Reply:
Fixed, I think :-) Take a look.